### DISCUSSION TITLE
AI 倉儲管理🏭

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#137] [General]
**新聞內容:**
工研院研發「AI 動態倉儲決策管理平台」（AiWT-DMP），從商品入倉開始，系統即啟動**丈量感測**功能，並依照大小安排適合儲放空間，省去人工丈量、安排儲位的誤差。消費者下單後，系統會**自動規劃最佳揀貨路徑**，進出貨準確度達 90%，透過穿梭車與垂直升降機跨樓層輸送，電子輔助揀貨系統（CAPS）再以光束與數字，導引揀貨人員挑出訂單商品放入揀貨籃。
此管理平台具有三大特色
一是出貨快，僅包裝到出貨階段用人工，高峰期更可提升 10 倍貨量。
二是密度高，倉庫採立體式設計，最高可達 14 層，儲位達 3～4 萬個。
三是決策準，分析因素，**預測消費習性**，並依關聯性建立模型。
深耕倉儲領域 30 年的新竹物流，對導入 AI 倉儲系統效率十分滿意。網購常因促銷或其他外在因素，突然變得熱賣，系統也能自動判斷商品熱銷程度或相關性，快速反應至適當儲位 
目前，工研院已攜手 Yahoo! 奇摩電商平台、新竹物流、漢錸科技，跨業合作建置東南亞首座「新世代智慧倉儲示範體系」。陳慧娟透露，也有臺商擬將此平台引進位於東南亞的倉庫，而團隊也規畫進一步與機械手臂結合，協助臺灣物流產業邁向智慧新時代。
**新聞心得:**
尤其一到節日或促銷活動，民眾就會瘋狂地下訂單，使用這項管理平台，不僅能使物流公司更快速的將貨物送達消費者家中，也讓整個工作流程去除大量人力成本，而且透過預測分析，還能提升效率，最令我驚訝的是，可放置的空間居然高達4萬個，究竟是台灣人太會消費，還是備著使用而已？！
**新聞來源:**
https://news.cnyes.com/news/id/4780255

**Comment by pocper1** [comment]
應該是台灣人太會消費了，雙11後，還有雙12...

**Comment by Timmy110403540** [comment]
這個平台讓網購業者真的方便了許多，而且可以同時因應台灣的「購物文化」。

**Comment by briankuo123** [comment]
之前忘記在哪個節日買東西，結果東西被物流運到不見，希望這個技術能避免這樣的情況發生。

**Comment by qeeeee39** [comment]
連貨運產業都漸漸地不需要人力了，AI愈來愈強OUO

**Comment by wh20502** [comment]
像那種一日出貨的可能都有使用到，不然哪有辦法請人(成本考量)每天在那邊盯訂單。

**Comment by ghost** [comment]
撇除機器可能凸槌出包的狀況 這是一個比起現在的模式還要更加便捷的模式呢 很期待

**Comment by ayakasanma** [comment]
現在的電商平台真的很需要呢，每逢節日折扣時那些物流都被塞滿，不僅便利商店被擠得水泄不通、常常購買的貨物都要兩三個例敗後才會收到。有了AI的幫助，相信塞車問題會好上不少！

**Comment by Shiba010** [comment]
這項技術看起來世界各地的電商平台都非常需要！

**Comment by Yunhans** [comment]
會不會之後送貨也不用人了

**Comment by LanningYang** [comment]
倉儲系統不只用於電商平台，在各工廠廠區的物料運用管理也相當適合，並可減少人力。
